ecosystem/sep-0010.md

```diff
@@ -24,10 +24,14 @@ The authentication flow is as follows:
 1. The client verifies that the transaction has an invalid sequence number 0.  This is extremely important to ensure the transaction isn't malicious.
 1. The client signs the transaction using the secret key(s) of signers for the user's Stellar account
 1. The client submits the signed challenge back to the server using [`token`](#token) endpoint
-1. The server gets the signers of the user's account
-1. The server verifies the signatures
-1. The server verifies the weight provided by the signers meets the required threshold(s), if any
-1. If the signature checks out, the server responds with a [JWT](https://jwt.io) that represents the user's session
+1. The server checks that the user's account exists
+1. If the user's account exists:
+  1. The server gets the signers of the user's account
+  1. The server verifies the signatures
+  1. The server verifies the weight provided by the signers meets the required threshold(s), if any
+  1. If the signature checks out, the server responds with a [JWT](https://jwt.io) that represents the user's session
+1. If the user's account does not exist (optional):
+  1. The server verifies the signature is correct for the master key of the account
```

Comment:
Do we want to note that there should not be any extra signatures other than the master key's?

Reply:
Good call. I added it in 70f6fd3. Can you 👀?